M5-R9

wchiquito opened the pull request:
Update README.md

wchiquito committed:
M5-R9
Update README.md

HugoNebreda committed:
M6-R2
Create tests to verify authentication

HugoNebreda committed:
M6-R2
Create basic authentication with no password check

HugoNebreda committed:
M6-R2
Verifies with DB password, but yet unencrypted

wchiquito committed:
M6-R2
Include crypt password

HugoNebreda committed:
M6-R2
New created users now encrypts password correctly

HugoNebreda committed:
M6-R2
Add tests to verfiy password encryption

wchiquito committed:
M6-R3-R4-R5
Complete milestone requirements

arielcamus commented on models/User.js:
```diff
@@ -0,0 +1,41 @@
+});
+
+UserSchema.pre('save', function(next) {
+  this.password = bcrypt.hashSync(this.password, bcrypt.genSaltSync(8), null);
```
Nice job going the extra mile here. We will add this as a requirement for future versions of the course :D

arielcamus reviewed (approved):
Great job in general. Let me know when you finish with Milestone 6 (authentication) and I will review it again.

wchiquito committed:
M6-R5
Added dynamic drop of collections

wchiquito committed:
M7-R1
Added partial ACL (Access Control Lists) for authorization

wchiquito committed:
M7-R1
Added partial ACL (Access Control Lists) for authorization - SOS

HugoNebreda committed:
M7-R1
    Added partial ACL (Access Control Lists) for authorization - after SOS

HugoNebreda committed:
M7-R1
Added full ACL (Access Control Lists) for event management and tests

wchiquito committed:
M7-R2
Implement an endpoint that returns all the events organized (created) by a given user

HugoNebreda committed:
M7-R2
Delete unnecessary test files. Caution you need to run tests twice to get it working

HugoNebreda committed:
M7-R2
Implement an endpoint that returns all the events organized (created) by a given user.  (Full)

HugoNebreda committed:
M7-R3
Implements an endpoint that allows a user to sign up to (RSVP) an event. (Completed)
